packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -225,7 +224,7 @@ export type Endpoints = {
   >;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
+// biome-ignore lint/suspicious/noExplicitAny: <explanation>
```

Comment:
would be good to add an explanation here, otherwise we set a bad precedence

Reply:
There should be, but as the earlier code comments didn't had that so we don't have it for now. 

It's good that biome enforce to provide some explanation if you disable any rule. I would suggest to add those explanations later as someone touches the code. Each occurrence may have different reason and I can't do it globally for all files.